tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionReplay.cs

```diff
@@ -65,25 +63,38 @@ private void InitSnapshotsSink()
             // Set up the snapshots sink.
             var snapshotSlicer = SnapshotSlicer.Create(debuggerSettings);
             _snapshotSink = SnapshotSink.Create(debuggerSettings, snapshotSlicer);
-            // TODO: respond to changes in exporter settings
```

Comment:
I would leave this warning, because it's still true 😉

Reply:
done in [716af21](https://github.com/DataDog/dd-trace-dotnet/pull/7859/commits/716af21292bc2f07b3fa58e2c7d714dcd402f0e9)